software/openvisualizer/SConstruct

```diff
@@ -134,6 +136,12 @@ AddOption('--simTopology',
     type      = 'string')
 runnerEnv['SIMTOPOLOGY'] = GetOption('simTopology')
 
+AddOption('--importTopo',
```

Comment:
I'm a bit confused, as you seem to declare a `pathTopo` parameter both in `SConstruct` and `openVisualizerApp.py`. I'm probably missing something.